### PR TITLE
398 explain final polish

### PR DESCRIPTION
**Pull request by ShiriMoran**
solves https://github.com/np-guard/vpc-network-config-analyzer/issues/398

### CODE REVIEW

**Review by adisos — COMMENTED** [review]
looks good

File: pkg/vpcmodel/nodesExplainability.go [hunk 1]
```diff
@@ -69,6 +69,10 @@ type Explanation struct {
 // ExplainConnectivity given src, dst and connQuery returns a struct with all explanation details
 // nil connQuery means connection is not part of the query
 func (c *VPCConfig) ExplainConnectivity(src, dst string, connQuery *common.ConnectionSet) (res *Explanation, err error) {
+	// we do not support multiple configs, yet
+	if c.IsMultipleVPCsConfig {
+		return nil, fmt.Errorf("multiple VPCs not supported by exlain mode, yet")
```
Comment (zivnevo) [inline pkg/vpcmodel/nodesExplainability.go:74]:
```suggestion
		return nil, fmt.Errorf("multiple VPCs not supported by explain mode, yet")
```

File: pkg/vpcmodel/printExplain.go [hunk 2]
```diff
@@ -132,8 +135,12 @@ func stringExplainabilityConnection(connQuery *common.ConnectionSet, src, dst En
 		resStr = fmt.Sprintf("The following connection exists between %v and %v: %v\n", src.Name(), dst.Name(),
 			conn.String())
 	} else {
-		resStr = fmt.Sprintf("Connection %v exists between %v and %v\n", conn.String(),
-			src.Name(), dst.Name())
+		properSubsetConn := ""
+		if !conn.Equal(connQuery) {
+			properSubsetConn = " (note that the connection is a proper subset of the queried one)"
```
Comment (zivnevo) [inline pkg/vpcmodel/printExplain.go:140]:
```suggestion
			properSubsetConn = " (note that not all queried protocols/ports are allowed)"
```

**Review by zivnevo — CHANGES_REQUESTED** [review]
Looks good. Just these two tiny comments

**Pull request merged**